README.md

```diff
@@ -149,56 +158,294 @@ This application will prompt users to fill in three or four games that they like
 │
 ├── reference/                        <- Any reference material relevant to the project
 │
-├── src/                              <- Source data for the project 
+├── src/                              <- Source data,python scripts for the project 
 │
 ├── test/                             <- Files necessary for running model tests (see documentation below) 
 │
 ├── app.py                            <- Flask wrapper for running the model 
-├── run.py                            <- Simplifies the execution of one or more of the src scripts  
+├── create_db.py                      <- Creates the video game database schema and populates data into it.
+├── Dockerfile                        <- Docker file to build image video_games and run app
+├── run.py                            <- Simplifies the execution of one or more of the src scripts - stores data in s3 bucket
 ├── requirements.txt                  <- Python package dependencies 
+├── run_mysql_client.sh               <- Runs interactive sql server based on host,username,password.
 ```
 
 ## Running the app
-### 1. Initialize the database 
+### 1.Clone this repository-development
+Run the following command:
+
+    git clone https://github.com/aditya9729/2020-msia423-gudal-gamexplorer.git
+    
+Once cloned run to switch to the development branch:
+	
+	git checkout development
+
+### 2.Data Acquisition
+Data sources - Kaggle static files:
+   * vgsales.csv link:https://www.kaggle.com/gregorut/videogamesales
+   * steam-200k.csv link:https://www.kaggle.com/tamber/steam-video-games
+   * metacritic_games.csv link:https://www.kaggle.com/skateddu/metacritic-games-stats-20112019
+ 
+Steps to follow:
+
+   * Download these files and unzip these files till you get comma separated value files.
+   * Store these files within ./data/external/.
+   * When you run the `create_db.py` file later, it will store cleaned data in ./data/clean/
+   * Please create the clean and external directory within the data directory.
+   * It should be ./data/clean and ./data/external
+   * If you want to recreate the file storage(when there is no external and clean directory) -
+    Run the following:
+   
+    cd ./data
+    mkdir external
+    mkdir clean
+    cd external
+   * Within external directory store the files acquired and rename them.
+        * Rename 'steam-200k.csv' to 'steam.csv'.
+        * Rename 'vgsales.csv' to 'sales.csv'.
+        * Rename 'metacritic_games.csv' to 'metacritic.csv'.
```

Comment:
Is there a way to not make the user do this step? It is prone to errors, and I'm sure you can take into account the original file names in your python script.

Reply:
Noted! Could you send the error message you are getting?